jsonnet/telemeter/client/kubernetes/kubernetes.libsonnet

```diff
@@ -39,17 +46,31 @@ local metricsPort = 8080;
       local clusterRole = k.rbac.v1.clusterRole;
       local policyRule = clusterRole.rulesType;
 
-      local rule = policyRule.new() +
-                   policyRule.withApiGroups(['']) +
-                   policyRule.withResources([
-                     'namespaces',
-                   ]) +
-                   policyRule.withVerbs(['get']);
+      local authenticationRule = policyRule.new() +
+                                 policyRule.withApiGroups(['authentication.k8s.io']) +
+                                 policyRule.withResources([
+                                   'tokenreviews',
+                                 ]) +
+                                 policyRule.withVerbs(['create']);
+
+      local authorizationRule = policyRule.new() +
+                                policyRule.withApiGroups(['authorization.k8s.io']) +
+                                policyRule.withResources([
+                                  'subjectaccessreviews',
+                                ]) +
+                                policyRule.withVerbs(['create']);
+
+      local coreRule = policyRule.new() +
+                       policyRule.withApiGroups(['']) +
+                       policyRule.withResources([
+                         'namespaces',
+                       ]) +
+                       policyRule.withVerbs(['get']);
```

Comment:
just a very small nit (can be done as a follow-up by me too): we should document this somewhere, as the "whoever can get namespaces, can scrape me" idiom seems to be ubiquitous.